db/warehouse/migrate/20251003192404_create_hud_lists.rb

```diff
@@ -0,0 +1,23 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class CreateHudLists < ActiveRecord::Migration[7.1]
+  def change
+    create_table :hud_list_items do |t|
+      t.string :list_name, null: false
+      t.string :method_name, null: false
+      t.string :list_number, null: false
+      t.string :label, null: false
+      t.integer :code, null: false
```

Comment:
I might be missing something but I don't think `code` should be an integer. These are strings right? ("2.02.6", etc). 

Suggest adding this and also

```suggestion
      t.string :code, null: false
```

Reply:
in `2.02.6`, `2.02.6` is the `list_number` (which is a string).  Codes are the keys:
```
    {
        0 => 'Emergency Shelter - Entry Exit',
        1 => 'Emergency Shelter - Night-by-Night',
        2 => 'Transitional Housing',
        3 => 'PH - Permanent Supportive Housing',
        4 => 'Street Outreach',
        6 => 'Services Only',
        8 => 'Safe Haven',
        9 => 'PH - Housing Only',
        7 => 'Other',
        10 => 'PH - Housing with Services (no disability required for entry)',
        11 => 'Day Shelter',
        12 => 'Homelessness Prevention',
        13 => 'PH - Rapid Re-Housing',
        14 => 'Coordinated Entry',
      }
```

Reply:
oh sorry, that wasn't a good example; many of them are integers but it looks like some of these have string keys that won't translate to integers.
* CoCCodes (key: "WY-500"...)
* VamcStationNumber (key: "636A8")
* "Race (key "RaceNone"..)

Reply:
Thanks!  Yes, you are correct.